corehq/toggles/__init__.py

```diff
@@ -2858,7 +2859,6 @@ def domain_has_privilege_from_toggle(privilege_slug, domain):
     'USH: Limit the location-owned cases that show up in a user\'s restore file',
     TAG_CUSTOM,
     namespaces=[NAMESPACE_DOMAIN],
-    help_link='https://dimagi.atlassian.net/wiki/spaces/USH/pages/2252210196/Prevent+Syncing+of+Lower+Level+Locations',  # noqa: E501
```

Comment:
This one should be fine to keep because it links to (in progress) documentation for the feature flag it is the `help_link` on.